Game/Src/Objects/sword.gd

```diff
@@ -1,4 +1,34 @@
 extends KinematicBody2D
 
+onready var sprite = $swordsprite
+onready var normalTexture = load('res://Assets/Objects/Sword.png')
```

Comment:
in src/ui/ToggleButton.gd we used the preload() function. No idea if one is better than the other but maybe you know :)

Reply:
There are slight differences but these do not really matter here. You could put preload back, it's supposedly a bit faster. I had some issues and tried load to solve it. It was not the cause but forgot to go back to preload.